backend/databasemodel/alembic/versions/95144b48054e/upgrade.sql

```diff
@@ -0,0 +1,12 @@
+-- This is a fix migration.
+-- Up to now, using just model.sql would result in the latter constraint names.
+-- Running migrations would result in the former names instead.
+-- This migration fixes the names to always have the same names as the model.sql.
+-- This prevents possible problems in future migrations and database operations.
+ALTER TABLE kooste.lipas_pisteet DROP CONSTRAINT IF EXISTS lipas_kohteet_piste_pk;
+ALTER TABLE kooste.lipas_viivat  DROP CONSTRAINT IF EXISTS lipas_kohteet_viiva_pk;
+ALTER TABLE kooste.lipas_pisteet DROP CONSTRAINT IF EXISTS lipas_pisteet_pk;
```

Comment:
Is it necessary here to drop the constraints with same name first and then add them back?

Reply:
@pavetsu14 Yes it is! The problem here is that we don't know what the constraint name is in the database. It might be either lipas_kohteet_piste_pk *or* lipas_pisteet_pk, depending on if the database was created from model.sql, or migrated from earlier version. This script has to work in both cases.

Therefore we want to make sure that there are no constraints in the database left. Then we can add the new constraint with the right name.

Reply:
It might be possible to make an sql script that checks the constraint names and runs different sql. This is a much easier way to do the same thing.

Reply:
Ah, I see! Thanks for clarification :)